### ISSUE TITLE
Feature request: option to disable tweening on links after graph change

### ISSUE THREAD

**Issue by cgibbs-gbg** [swimlane/ngx-graph#182]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
Currently, when the structure for a graph changes, the links in the graph tween relatively smoothly to their new positions, so that the viewer can see what actually changed in the graph without a jarring change. This is most likely expected behavior for almost all ngx-graph users.

**Expected behavior**
When given a boolean flag to disable link tweening, graph changes would make all links redraw instantly, rather than doing the interpolated movement to their new location.

**Reproduction of the problem**
This behavior can be seen in the ngx-graph demo here, when real-time data is enabled: https://swimlane.github.io/ngx-graph/

**What is the motivation / use case for changing the behavior?**
Our project implements ngx-graph to show a relationship tree between entities using (lowercase) angular lines; clicking on nodes refocuses the clicked entity to be the focus (not always the root) of the tree, at which point the (lowercase) angular lines exhibit the tweening behavior, which doesn't look very good with the curve algorithm we've chosen (d3.shape.curveLinear). It would be better for our case if the lines didn't tween, and just drew the new relationship instantly.

* **ngx-graph version:** 4.3.0
Side-note: we're in the process of upgrading to the latest Angular so that we can use the latest version of ngx-graph (and other libraries, of course); if this is something that's available in a newer version, or will only be added to a newer version, that's fine. There are no expectations that this feature would be retroactively added to 4.x of ngx-graph.

* **Angular version:** 5.1.0

As a bit of a post-script, ngx-graph is really great! It has basically everything we needed in order to display the information we want in a way that looks nice, so you've done a very good job with it.

**Comment by assafsun** [comment]
So you want to disable the links animation when the graph changes?

**Comment by cgibbs-gbg** [comment]
@assafsun That's correct.

**Comment by marjan-georgiev** [comment]
Version 6 supports the `animate` input, which enables/disables animations on the graph.

**Comment by assafsun** [comment]
Is it possible to upgrade to version 6?

**Comment by cgibbs-gbg** [comment]
It is possible to upgrade eventually, but it's a long-term goal at the moment, due to project constraints. In the meantime, I can at least give this as yet another reason to bump Angular. Thanks for the response!

**Comment by assafsun** [comment]
Created a PR for version 5

**Comment by assafsun** [comment]
Should be available on latest version 5 - 5.6.0
